CHANGELOG.md

```diff
@@ -19,7 +31,6 @@ v1.10.0
 ### Removed
 - `resolutions` value for `fozzie-colour-palette` no longer needed
 
-
```

Comment:
Mega anal comment from me – all the CHANGELOG's have two linebreaks between entries, so should follow that convention here too.